Apps/W1/HybridGP/app/app.json

```diff
@@ -29,7 +29,13 @@
                              "name":  "Sales and Inventory Forecast",
                              "publisher":  "Microsoft",
                              "version":  "24.0.0.0"
-                         }
+                        },
```

Comment:
I just checked with Nikola. Two reasons that he gave.

1. Easier to maintain the feature.
2. Since it is modular, it can be installed or uninstalled by the customer.

I imagine it being much more complicated to get the Base app changed.